python/lsst/ts/standardscripts/maintel/m1m3/raise_m1m3.py

```diff
@@ -0,0 +1,69 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["RaiseM1M3"]
+
+import time
+
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+from lsst.ts import salobj
+
+
+class RaiseM1M3(salobj.BaseScript):
+    """Raise M1M3 mirror.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    """
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Raise M1M3")
+
+        mtcs_usage = None
+
+        self.mttcs = MTCS(self.domain, intended_usage=mtcs_usage, log=self.log)
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
```

Comment:
Don't we need a schema even if it has no parameters? Not sure.

Reply:
I haven't seen a schema in any of the zero-parameters scripts but I agree it would be nice to see some information in the configuration screen rather than **"WARNING: failed to automatically determine site"**. Any thoughts @tribeiro?

Reply:
If the Script does not have configuration parameters the `get_schema` should return `None`.. so, it is all good.